source/getting-started/managing-dependencies.md

```diff
@@ -0,0 +1,263 @@
+directories. The following example scenarios illustrate how this works.
+
+#### Javascript Assets
+
+##### Standard Non-AMD Asset
+
+Provide the asset path as the first and only argument:
+
+```Brocfile.js
+app.import('bower_components/moment/moment.js');
+```
+
+From here you would use the package as specified by it's documentation, usually a global variable.
+In this case it would be:
+
+```js
+import Ember from 'ember';
+/* global moment */
+// No import for moment, it's a global called `moment`
+
+// ...
+var day = moment('Dec 25, 1995');
+```
+
+_Note: Don't forget to make JSHint happy by adding a `/* global MY_GLOBAL */` to your module, or
+by defining it within the `predefs` section of your `.jshintrc` file._
+
+##### Standard AMD Asset
+
+Provide the asset path as the first argument, and the list of modules and exports as the second:
+
+```Brocfile.js
+app.import('bower_components/ic-ajax/dist/named-amd/main.js', {
+  exports: {
+    'ic-ajax': [
+      'default',
+      'defineFixture',
+      'lookupFixture',
+      'raw',
+      'request',
+    ]
+  }
+});
+```
+
+To use this asset in your app, import it.
+For example, with `ic-ajax`, when to use `ic.ajax.raw`:
+
+```js
+import { raw as icAjaxRaw } from 'ic-ajax';
+//...
+icAjaxRaw( /* ... */ );
+```
+
+##### Environment Specific Assets
+
+If you need to use different assets in different environments, specify an object as the first parameter. That object's key should be the environment name, and the value should be the asset to use in that environment.
+
+```Brocfile.js
+app.import({
+  development: 'bower_components/ember/ember.js',
+  production:  'bower_components/ember/ember.prod.js'
+});
+```
+
+If you need to import an asset in one environment but not import it or any alternatives in other environments then you can wrap `app.import` in an `if` statement.
+
+```Brocfile.js
+if (app.env === 'development') {
+  app.import('vendor/ember-renderspeed/ember-renderspeed.js');
+}
+```
+
+##### Customizing a built-in Asset
+
+This is somewhat non-standard and discouraged, but suppose that due to a requirement in your application that you need to use the full version of
+Handlebars even in the production environment.  You would simply provide the path to the `EmberApp` constructor:
+
+```Brocfile.js
+var app = new EmberApp({
+  vendorFiles: {
+    'handlebars.js': {
+      production: 'bower_components/handlebars/handlebars.js'
+    }
+  }
+});
+```
+
+Alternatively, if you want to exclude the built-in asset from being
+automatically included in `vendor.js`, you can set its value to `false`:
+
+```Brocfile.js
+var app = new EmberApp({
+  vendorFiles: {
+    'handlebars.js': false
+  }
+});
+
+```
+
+_Note: The built-in assets are required dependencies needed by the environment
+to run your app. If you use the above method to specifically exclude
+some, you should still be including them in some other way._
+
+##### Test Assets
```

Comment:
How is this different than environment-specific assets, covered above? Why wouldn't you just use the same syntax as above?